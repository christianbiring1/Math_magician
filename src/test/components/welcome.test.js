import React from 'react';
import renderer from 'react-test-renderer';
import Welcome from '../../components/welcome';

it('matches Welcome snapshot', () => {
  const tree = renderer.create(
    <Welcome />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
