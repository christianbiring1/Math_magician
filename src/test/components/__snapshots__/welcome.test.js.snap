// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches Welcome snapshot 1`] = `
<div
  className="welcome-container"
>
  <h1>
    Welcome to our page
  </h1>
  <div
    className="introduction"
  >
    <p>
      A calculator is a device that performs arithmetic operations on numbers. The simplest calculators can do only addition, subtraction, multiplication, and division. More sophisticated calculators can handle exponential operations, roots, logarithms, trigonometric functions, and hyperbolic functions. Internally, some calculators actually perform all of these functions by repeated processes of addition.
    </p>
    <p>
      As calculators became more advanced during the 1970s, they became able to make computations involving variables (unknowns). These were the first personal computers. Todays personal computers can still perform such operations, and most are provided with a virtual calculator program that actually looks, on screen, like a handheld calculator. The buttons are actuated by pointing and clicking.
    </p>
  </div>
</div>
`;
